lib/main.js

```diff
@@ -163,6 +238,10 @@ function loadLibraries() {
                                      [ctypes.char.ptr,
                                       ctypes.int,
                                       ctypes.voidptr_t]);
+  PK11_CheckUserPassword = declareFunction("PK11_CheckUserPassword", nss3,
+                                     [SECStatus,
```

Comment:
OK, it makes more sense this way. I copied allignment from PR_ErrorToString. Will align it correctly on second attempt.